docker-compose/regtest/eth.yml

```diff
@@ -5,6 +5,50 @@ services:
     command: -p=8545 -h=0.0.0.0 -m="piece effort bind that embrace enrich remind powder sudden patient legend group"
     volumes:
       - ${BLOCKCHAIN_PATH}/ethereumgo:/root/.ethereum
+    healthcheck:
+      test: ["CMD-SHELL", "wget -q -O - http://localhost:8545 --post-data '{\"jsonrpc\":\"2.0\",\"method\":\"net_listening\",\"params\": [],\"id\":1}'"]
+      interval: 5s
+      timeout: 15s
+      retries: 5
+
+  postgres-regtest:
+    build: ../../packages/nodechain-ethereum-indexer
+    image: nodechain-ethereum-indexer
+    volumes:
+      - ${BLOCKCHAIN_PATH}/postgres:/var/lib/postgresql/data
```

Comment:
Maybe a good improvement could be to mount regtest, testnet, and mainnet in different routes? i.e: bitcoincore is always mount in the same path because it creates inside its volume a specific folder for each network. In this case, I think if someone has mainnet blockchain sync and run accidentally testnet network his entire database can be corrupted.